### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by Eleoes**

### CODE REVIEW

File: main_app/templates/home.html [hunk 1]
```diff
@@ -1,20 +1,19 @@
...
+            <a href="{% url 'signup' %}" class="btn green darken-4">Signup</a>
         </div>
-        <a class="carousel-item" href="#two!"><img src="{% static "/imgs/people/00109-2609278882.png"%}"></a>
-        <a class="carousel-item" href="#three!"><img src="{% static "/imgs/people/00088-655689592.png"%}"></a>
-        <a class="carousel-item" href="#four!"><img src="{% static "/imgs/people/00018.jpg"%}"></a>
-        <a class="carousel-item" href="#five!"><img src="{% static "/imgs/people/00008.png"%}"></a>
-        <a class="carousel-item" href="#six!"><img src="{% static "/imgs/people/00016.png"%}"></a>
+    </div>
+    <div class="col s6 img-container">
+        <img src="https://www.iwantplants.co.uk/wp-content/themes/iwantplants/assets/images/floating-plant-5.png" alt="plants"/>
```
Comment (williethewolf) [inline main_app/templates/home.html]: Not hot linking images in other hosts

File: main_app/static/imgs/plants/general-use/home-image.png:Zone.Identifier [hunk 2]
```diff
@@ -0,0 +1,4 @@
+[ZoneTransfer]
```
Comment (williethewolf) [inline main_app/static/imgs/plants/general-use/home-image.png:Zone.Identifier]: remove the zone.identifier for the images or add **/*Zone.Identifier to .gitignore

**Review by williethewolf — CHANGES_REQUESTED** [review]
images: Let's make sure we are using assets which are royalty free, a CC0 attribution, in the public domain or self generated.

File: main_app/templates/home.html [hunk 3]
```diff
@@ -5,11 +5,11 @@
     <div class="col s6">
         <div class="home-content">
             <span class="app-title">PLANT PARENTHOOD</span>
-            <p>Nurturing your home, one plant a time</p>
-            <a href="{% url 'signup' %}" class="btn green darken-4">Signup</a>
+            <p>Our aim is to inspire and provide you the training and knowledge you need to be a proud plant parent</p>
+            <a href="{% url 'signup' %}" class="btn green darken-4 signup">Signup</a>
```
Comment (williethewolf) [inline main_app/templates/home.html:9]: Sign-up should go back to the base template. Not logged-in users can land on more pages than home. The public database and corresponding detail pages for example should be a landing page from search engines.

Removing a CTA from the base template make it that you have to manually implement them on every possible landing page template

File: main_app/templates/base.html [hunk 4]
```diff
@@ -16,37 +16,17 @@
...
-                <li><a href="{% url 'plant_add' %}">Add a plant</a></li>
-                {% if request.user.is_active and request.user.is_staff %}
-                <li><a href="{% url 'plant_create' %}">Create plant entry in DB</a></li>
-                {% endif %}
-            </ul>
-            <ul id="viewdropdown" class="dropdown-content">
-                <li><a href="{% url 'plants_index' %}">My plants</a></li>
-                <li><a href="{% url 'plantsDB_index' %}">Plants DB</a></li>
-            </ul>
-            <div class="parallax-container">
```
Comment (williethewolf) [inline main_app/templates/base.html:30]: Parallax effect removed - please remove corresponding commented code from script at the bottom unless usign parallax container somewhere else.

File: main_app/templates/base.html [hunk 5]
```diff
@@ -16,37 +16,17 @@
     </head>
     <body>
         <header class="">
-            <ul id="adddropdown" class="dropdown-content">
```
Comment (williethewolf) [inline main_app/templates/base.html]: dropdown display links removed. These lines are needed for the line from 32 to 40 to work

**Review by williethewolf — CHANGES_REQUESTED** [review]
Check out the comments. Functionality as of now is broken.

File: main_app/static/imgs/plants/general-use/home-image.png:Zone.Identifier (hunk 2 above)
Comment (Eleoes) [reply main_app/static/imgs/plants/general-use/home-image.png:Zone.Identifier]: Removed

File: main_app/templates/home.html (hunk 1 above)
Comment (Eleoes) [reply main_app/templates/home.html]: Removed hot link, used one of the AI ones instead for now.

File: main_app/templates/home.html (hunk 3 above)
Comment (Eleoes) [reply main_app/templates/home.html:9]: Signup link is back on the navbar. Navbar is now responsive but pending testing dropdown menu.

File: main_app/templates/base.html (hunk 4 above)
Comment (Eleoes) [reply main_app/templates/base.html:30]: Parallax effect removed including script.

File: main_app/templates/base.html (hunk 5 above)
Comment (Eleoes) [reply main_app/templates/base.html]: Dropdown display links are back. Pending integration with responsive navbar code.

**Pull request closed without merging**